integration_tests/e2e/goal/addNote.cy.ts

```diff
@@ -57,8 +57,7 @@ context('Add a note', () => {
       .setStepTitle('Book French course')
       .submitPage()
 
-    const addNotePage = Page.verifyOnPage(AddNotePage)
-    addNotePage.isForPrisoner(prisonNumber)
+    Page.verifyOnPage(AddNotePage)
```

Comment:
All calls to `isForPrisoner` on all the Goal related pages are now removed, because they check the prisoner's prison number in the Prisoner Summary bar .... which has now been removed